index.bs

```diff
@@ -1300,27 +1300,60 @@ Issue: Define how this works.
 
 # User Contexts # {#user-contexts}
 
-A <dfn>user context</dfn> represents a collection of top-level navigables within
-a [=remote end=]. User contexts have a <dfn>user context id</dfn>, which is a
-string set upon the user context creation. The [=default user context=]'s [=user
-context id|id=] is <code>"default"</code>. A [=remote end=] always contains one
-[=user context=] referred as the <dfn>default user context</dfn>.
-
-Each top-level navigable belongs to exactly one [=user context=], and child
-navigables belong to the same [=user context=] as their parent. Each navigable's
-[=user context=] is the [=user context=] the navigables belongs to.
-
-A [=user context=] without navigables belonging to it is an <dfn>empty
-user context</dfn>.
-
-Each [=user context=] represents an additional [=storage key=], so that no
-storage state is shared between different [=user context|user contexts=].
+A <dfn>user context</dfn> represents a collection of zero or more
+[=/top-level traversables=] within a [=remote end=]. Each [=user context=] has
+an associated [=storage partition=], so that [=remote end=] data is not shared
+between different [=user contexts=].
 
 Note: The infra spec uses the term "user agent" to refer to the same concept as
 [=user context|user contexts=]. However, this is not compatible with usage of
 the term "user agent" to mean the entire web client with multiple [=user
 context|user contexts=]. Although this difference is not visible to web content,
-it is observed via WebDriver, so we avoid reusing existing terminology.
+it is observed via WebDriver, so we avoid using this terminology.
+
+Issue: Unclear if this is the correct way to define this.
```

Comment:
Does this refer to the entire definition or the next sentence?

Reply:
The entire definition.